docs/sources/shared/flow/reference/components/rule-block.md

```diff
@@ -33,4 +33,4 @@ Here's a list of the available actions, along with a brief description of their
 * `labelkeep` - Matches `regex` against all label names. Any labels that don't match are removed from the metric's label set.
 
 Finally, note that the regex capture groups can be referred to using either the
-`$1` or `$${1}` notation.
+`$1` or `${1}` notation.
```

Comment:
@rfratto and I were discussing whether to use Google's placeholder suggestion (https://developers.google.com/style/placeholders#placeholders-in-inline-text) in [this thread](https://raintank-corp.slack.com/archives/C02GSU8SHBN/p1666123999924289?thread_ts=1666123989.274949&cid=C02GSU8SHBN). In that case, we would want to use something like: 

```suggestion
`$CAPTURE_GROUP_NUMBER` or `${CAPTURE_GROUP_NUMBER}` notation.
```
I leave the decision up to you all...

Reply:
It's a GitHub action which adds issues and PRs to our project board, but it doesn't work for PRs from forks. I need to remove the PR automation; you can ignore it failing for now.